src/data/articles.yaml

```diff
@@ -0,0 +1,19 @@
+- title: Wiki info about the city of Portland, OR
+  author: Author Name
+  date: May 5th, 2016
+  excerpt: This is a short excerpt.
+  body: |
+    <p>Portland (/ˈpɔːrtlənd/) is the largest city in the U.S. state of Oregon and the seat of Multnomah County. It is located in the Willamette Valley region of the Pacific Northwest, at the confluence of the Willamette and Columbia rivers. The city covers 145 square miles (376 km²) and had an estimated population of 619,360 in 2014,[7][9] making it the 28th most populous city in the United States. Approximately 2,348,247 people live in the Portland metropolitan statistical area (MSA), the 24th most populous MSA in the United States. Its Combined Statistical Area (CSA) ranks 17th with a population of 3,022,178. Roughly 60 percent of Oregon's population resides within the Portland metropolitan area.[10]</p>
+
+    <p>Named after the city on the coast of Maine (which was named after the English Isle of Portland), the Oregon settlement began to be populated in the 1830s near the end of the Oregon Trail. Its water access provided convenient transportation of goods, and the timber industry was a major force in the city's early economy. At the turn of the 20th century, the city had developed a reputation as one of the most dangerous port cities in the world, a hub for organized crime and racketeering. After the city's economy experienced an industrial boom during World War II, its hard-edged reputation began to dissipate. Beginning in the 1960s,[11] Portland became noted for its growing liberal political values, and the city has earned a reputation as a bastion of counterculture, a view which has proceeded into the 21st century.[12] According to a 2009 Pew Research Center study, Portland ranks as the 8th most popular American city, based on where people want to live.[13]</p>
+
+- title: Wiki info about the state of Oregon
+  author: Author Name
+  date: May 12th, 2016
+  excerpt: Second article excerpt.
+  body: |
+    <p>Oregon (Listeni/ˈɔːrᵻɡən/ awr-ə-gən)[7] is a state in the Pacific Northwest region of the United States. Oregon is bordered on the west by the Pacific Ocean, on the north by Washington, on the south by California, on the east by Idaho, and on the southeast by Nevada. The Columbia River delineates much of Oregon's northern boundary, and the Snake River delineates much of the eastern boundary. The parallel 42° north delineates the southern boundary with California and Nevada. It is one of only three states of the contiguous United States to have a coastline on the Pacific Ocean, and the proximity to the ocean heavily influences the state's mild winter climate, despite the latitude.</p>
```

Comment:
> It's got a lot of bracketed footnotes, pronunciation guides, etc.

@tylersticka Yeah, after I pieced it together I wondered if this was the best sample content. I'll look around to see if I find something better. Otherwise, lorem ipsum it is.